terraform/utils.go

```diff
@@ -25,6 +25,15 @@ func isValidInputsFile(fileName string) bool {
 	return false
 }
 
+func isValidLocalsFile(fileName string) bool {
+	for _, fileExt := range VALID_TERRAFORM_FILES {
+		if strings.HasSuffix(fileName, fileExt) {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
isn't this the same as the `isValidTerraformFile` below? is there something different I'm missing?

Reply:
it is indeed the same :)